disnake/message.py

```diff
@@ -1371,8 +1383,18 @@ def system_content(self):
             return self.reference.resolved.content  # type: ignore
 
         if self.type is MessageType.guild_invite_reminder:
+            # todo: determine if this should be the owner content or the user content
```

Comment:
This should be fine for now, anything else would have to rely on checking guild ownership and permissions.

Reply:
I still would like to leave this todo in here.